### ISSUE TITLE
Review Heatmap Error

### ISSUE THREAD

**Issue by akmfrench** [glutanimate/review-heatmap#35]
#### Problem description or Feature request

*Please describe the issue concisely in here. In case of an error: Walk us through the steps you took to get there. What happened? What did you expect to happen?*

Downloaded anki 2.1 heat map folder, moved folder to addons21 folder.
Upon restarting anki, I get an error message, copied below.

#### Checklist

*Please replace the space inside the brackets with an **x** if the following sentences apply to you. Feel free to disregard some of these if you're posting a feature request rather than a bug report.*

 - [x ] I restarted Anki to see if it helps
 - [ ] I verified that I use the latest version of the add-on by redownloading it from AnkiWeb
 - [x ] I verified that I use the latest version of Anki by checking at https://apps.ankiweb.net#download
 - [ ] I tried to disable other add-ons to see if there are any interactions present
 - [x ] I checked if anyone reported this problem before by looking through the issue reports. I also checked to see if there is a section about known issues in the add-on description, documentation, or README.
 - [ ] My issue is solved when holding shift while starting Anki.
 
#### Information about your Anki Setup

<details>

*Please open Anki, go to Help → About, look-up two lines, starting with `Version:` and `Qt` and copy-paste them below:*

```
Version 2.1.11 (3cf770c7)
Qt 5.12.1 PyQt 5.11.3
```

*Please fill in your OS details (Linux/Mac/Windows, which version):*

```
OS: Mac 10.14.4
```

*Please open Anki, go to Tools → Add-ons, take a screenshot of your installed add-ons, and paste it below:*
 
![image](https://user-images.githubusercontent.com/49712125/56272027-55b0d280-60a6-11e9-8ac8-763231501ac7.png)

</details>

#### Error message (if any)

<details>

*If you got an error message, please copy and paste it below:*

An add-on you installed failed to load. If problems persist, please go to the Tools>Add-ons menu, and disable or delete the add-on.

When loading 'review_heatmap':
Traceback (most recent call last):
  File "aqt/addons.py", line 71, in loadAddons
  File "/Users/MyName/Library/Application Support/Anki2/addons21/review_heatmap/__init__.py", line 41, in <module>
    from . import main  # noqa: F401
  File "/Users/MyName/Library/Application Support/Anki2/addons21/review_heatmap/main.py", line 40, in <module>
    from .gui.options import initializeOptions
  File "/Users/MyName/Library/Application Support/Anki2/addons21/review_heatmap/gui/options.py", line 46, in <module>
    from ..libaddon.gui.dialog_options import OptionsDialog
  File "/Users/MyName/Library/Application Support/Anki2/addons21/review_heatmap/libaddon/gui/dialog_options.py", line 44, in <module>
    from ..platform import PLATFORM
ImportError: cannot import name 'PLATFORM'

```python"

</details>

**Comment by akmfrench** [comment]
![image](https://user-images.githubusercontent.com/49712125/56272647-a07f1a00-60a7-11e9-88c1-f46503e0561f.png)
s  

I downloaded this file review-heatmap-release-v0.7.0-beta.1-anki21 
and drug it into the addons21 folder

**Comment by glutanimate** [comment]
Hi @akmfrench,

Thanks for the report. Unfortunately I can't reproduce the issue here. From the error message it seems like something went wrong while copying the data into the add-ons folder (the code segments quoted in the message no longer exist, so somehow there are files from an earlier add-on release in there).

Please delete the review_heatmap folder in your ad-ons directory, then download [this zip file](https://github.com/glutanimate/review-heatmap/releases/download/v0.7.0-beta.1/review-heatmap-release-v0.7.0-beta.1-anki21.zip) and go through the [installation steps](https://github.com/glutanimate/review-heatmap/releases/tag/v0.7.0-beta.1) again.

**Comment by akmfrench** [comment]
@glutanimate 

Thank you for the response! I tried it all again with no luck. Even tried deleting Anki 2.0 to see if that helped at all. Will there be an official release for this at any point/ any eta on that? 

I appreciate all your work!

**Comment by lucascp2004** [comment]
I got this error today
`When loading 'review_heatmap':
Traceback (most recent call last):
  File "aqt/addons.py", line 80, in loadAddons
  File "/home/lucas/.local/share/Anki2/addons21/review_heatmap/__init__.py", line 41, in <module>
    from . import main  # noqa: F401
  File "/home/lucas/.local/share/Anki2/addons21/review_heatmap/main.py", line 40, in <module>
    from .gui.options import initializeOptions
  File "/home/lucas/.local/share/Anki2/addons21/review_heatmap/gui/options.py", line 46, in <module>
    from ..libaddon.gui.dialog_options import OptionsDialog
  File "/home/lucas/.local/share/Anki2/addons21/review_heatmap/libaddon/gui/dialog_options.py", line 44, in <module>
    from ..platform import PLATFORM
ImportError: cannot import name 'PLATFORM'`

`Version 2.1.14 (7b93e985)
Qt 5.12.1 PyQt 5.11.3`
![Screenshot_2019-07-22_18-17-18](https://user-images.githubusercontent.com/43997650/61666073-fd90fa00-acac-11e9-9f65-e9233dbf9c8a.png)
webcodings and bleach are python librarys
lucas is my addon that use these librarys

**Comment by lucascp2004** [comment]
When i remove all addons still don't work

**Comment by glutanimate** [comment]
This was an issue with Anki's add-on system that is now fixed in the 2.1.15 beta. You can download the beta [here](https://apps.ankiweb.net/downloads/beta/).  For more information please also see #43.

(This issue report came first, but since the other one now contains more in-depth discussion about the problem, I will be marking this one as the duplicate)
